.gitignore

```diff
@@ -86,6 +86,17 @@ platform/android/java/*/libs/
 *.bc
 platform/javascript/node_modules/
 
+# Vita
+temp-build/
+*.vpk
+
+# Switch
+*.nacp
+*.nca
```

Comment:
Why? We don't generate these.

Reply:
It's useful to have any platform-specific binaries in the list in case they somehow end up there. I didn't expect to see `.nacp` files, but I found some in the repo. So I'm just being thorough.